pkg/controller/v1alpha1/inferencegraph/controller_test.go

```diff
@@ -163,6 +164,23 @@ var _ = Describe("Inference Graph controller test", func() {
 													v1.ResourceMemory: resource.MustParse("100Mi"),
 												},
 											},
+											ReadinessProbe: &v1.Probe{
```

Comment:
This very same code is used in different places, maybe we can move to a utils package or something a reuse it.

Reply:
This is used in tests for assert against the actual and expected knative service. I believe it is better to leave it as it is. WDYT ?

Reply:
I meant the probes are the same for all the places to instantiate it.
Maybe move this part to a common env or func.

Reply:
done